pkg/tfbridge/info.go

```diff
@@ -177,6 +177,9 @@ type SchemaInfo struct {
 
 	// this will make the parameter as computed and not allow the user to set it
 	MarkAsComputedOnly *bool
+
+	// the deprecation message for the property
+	Deprecated string
```

Comment:
Can we name this `DeprecationMessage` for consistency with `ResourceInfo.DeprecationMessage`?